### PR TITLE
Bugfixes and features (task #2857)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

File: src/FieldHandlers/DecimalFieldHandler.php [hunk 1]
```diff
@@ -16,14 +16,16 @@ class DecimalFieldHandler extends BaseFieldHandler
      */
     public function renderValue($table, $field, $data, array $options = [])
     {
-        $result = filter_var($data, FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION);
+        $result = (float)filter_var($data, FILTER_SANITIZE_NUMBER_FLOAT, FILTER_FLAG_ALLOW_FRACTION);
```
Comment (mamchenkov) [inline src/FieldHandlers/DecimalFieldHandler.php]: @georgeconstantinou can you please also do a similar fix on the Integer field.

Comment (georgeconstantinou) [inline src/FieldHandlers/DecimalFieldHandler.php]: @mamchenkov e85dae5 6af67d7

Comment (mamchenkov) [inline src/FieldHandlers/DecimalFieldHandler.php]: @georgeconstantinou ... and unit tests :)

Comment (georgeconstantinou) [inline src/FieldHandlers/DecimalFieldHandler.php]: @mamchenkov 5caa20c

**Pull request merged**